### DISCUSSION TITLE
Is this safe

### DISCUSSION THREAD

**Discussion by Targink321** [Raphire/Win11Debloat#170] [Q&A]
I checked and read thru the code but I’m kinda dumb so is this safe I think so tho

**Comment by Targink321** [comment]
Okay I just checked and why is the script you download via power shell commands hash different then the one you get from Manual  download

**Comment by Raphire** [comment]
The powershell command that downloads the script for you does so directly from github, so it should give you the same version as the one you'd get from manually downloading it. 

What are you hashing? The script file itself?

**Comment by Targink321** [comment]
Yes the ps1

**Comment by Targink321** [comment]
It’s a different hash 😭😭😭😭 seems safe tho

**Comment by Targink321** [comment]
So I both downloaded from GitHub and using powershell (not the one provided ) they have the same hash but the quick methods one it had a diff hash  I don’t have a screen shot btw and yes I used the same algorithm

**Comment by Targink321** [comment]
Tho if you wanted to steal ppls stuff it would prob be in the manual ver too so I don’t think you are trying to hack anyone

**Comment by Raphire** [comment]
> It’s a different hash 😭😭😭😭 seems safe tho

I am getting the same hash using the `Get-FileHash` function built into Powershell for the `Win11Debloat.ps1` file downloaded manually and the one downloaded by using the `Quick method`.  The scriptfiles as downloaded by the`Quick method` can be found at `%temp%/Win11Debloat`.

~~The SHA256 hash I am getting from the current version of the `Win11Debloat.ps1` file is `60F8E5B30C5489E3D2E4AB794F380F73885C867016632DDFADBAB1A95B861498` for reference.~~

EDIT:
Please note that the hash will change every time the file is updated.
